src/pynwb/ndx_external_resources/resources.py

```diff
@@ -0,0 +1,11 @@
+from hdmf.common import ExternalResources as hdmf_ExternalResources
+from pynwb import register_class, get_type_map
+from hdmf.utils import docval, get_docval, call_docval_func
+
+class ExternalResources(hdmf_ExternalResources):
+
+    @docval(*get_docval(hdmf_ExternalResources.__init__))
+    def __init__(self, **kwargs):
+        call_docval_func(super().__init__, kwargs)
+        # super().__init__(keys, resources, entities, objects, object_keys)
+        self.type_map = kwargs['type_map'] or get_type_map()
```

Comment:
Can you test what happens when a user passes in their own TypeMap to this constructor?

Reply:
I tested it. It works fine.